allennlp/tests/common/plugins_util.py

```diff
@@ -1,9 +1,12 @@
 from contextlib import contextmanager
 
-from pip._internal.cli.main import main as pip_main
-
 from allennlp.common.util import ContextManagerFunctionReturnType, PathType, push_python_path, pushd
 
+try:
```

Comment:
Please can you add a comment about why this try except block is here? Just something like "pip moved this entrypoint somewhere between version 19-20, so we check both places, because it's likely that a user does not fix their version of pip". 

Also, are these the only two possible locations for this function? I'm trying to understand if it would be sensible to have the second import (in the except block) also with a try/catch, so we can recommend that someone upgrade their pip version if it does crash.

Reply:
> Also, are these the only two possible locations for this function? I'm trying to understand if it would be sensible to have the second import (in the except block) also with a try/catch, so we can recommend that someone upgrade their pip version if it does crash.

For pip<10, yes. But that's kinda old. Do you think we should still do so?

Reply:
nah, that's fine then 👍